data/state/peerAccount_test.go

```diff
@@ -563,3 +563,93 @@ func TestPeerAccount_DecreaseValidatorSuccessRateWithJournal(t *testing.T) {
 	assert.Equal(t, 1, journalizeCalled)
 	assert.Equal(t, 1, saveAccountCalled)
 }
+
+func TestPeerAccount_IncreaseNumSelectedInSuccessBlocks(t *testing.T) {
```

Comment:
add test to the 4 new journalizing elements and revert on them. 2 in accounts, 2 in peer accounts

Reply:
done